custom_reports/columns.rst

```diff
@@ -96,315 +96,297 @@ The table below lists core dimensions that may be used in queries.
 
 .. table:: Base Dimensions
 
-    +--------------------------------------+-----------------------------------+----------+--------+--------------------------------------------------------------------------------------------------+
-    |            Dimension Name            |             Column ID             |   Type   |Nullable|                                            Enum Table                                            |
```

Comment:
@nfpp It's standard upstream merge. Is there a reason why v10.1 doesn't use updated columns, or can I merge it safely to match those from v10.0?
v10.0: https://developers.piwik.pro/en/10.0/custom_reports/columns.html
v10.1: https://developers.piwik.pro/en/10.1/custom_reports/columns.html

Reply:
@RomanSek  I don't understand this diff, locally I copied this table to separate files from those two branches and do `diff 10.0.txt 10.1.txt` and there are no changes. Please check after the merge that it will stay this way.